From a051210cb5e37c6512e9a3ecc051337007bc0b9a Mon Sep 17 00:00:00 2001
From: Yu Yue <yuyue@mprc.pku.edu.cn>
Date: Mon, 31 Oct 2011 11:29:05 +0800
Subject: [PATCH 048/641] UniCore64: Add function show_regs

Fixed error:
hid-input.c:(.text+0x1a754): undefined reference to `show_regs'

Signed-off-by: Yu Yue <yuyue@mprc.pku.edu.cn>
---
 arch/unicore64/kernel/process.c |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/unicore64/kernel/process.c b/arch/unicore64/kernel/process.c
index 4316735..eb955eb 100644
--- a/arch/unicore64/kernel/process.c
+++ b/arch/unicore64/kernel/process.c
@@ -108,3 +108,13 @@ int dump_fpu(struct pt_regs *regs, elf_fpregset_t *fp)
 	BUG();
 	return 0;
 }
+
+/**
+ * show_regs() -
+ * @regs:
+ */
+void show_regs(struct pt_regs *regs)
+{
+	/* FIXME */
+	BUG();
+}
-- 
1.7.9.5

